subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1277,6 +1277,11 @@ int ztls_accept_ctx(struct net_context *parent, struct sockaddr *addr,
 	}
 
 	child = k_fifo_get(&parent->accept_q, K_FOREVER);
+	if (child == NULL) {
```

Comment:
Why would `k_fifo_get` return a NULL if it's called with `K_FOREVER` timeout? AFAIK it's not possible, unless there's some bug in `k_fifo_get`. @jukkar Or do I miss something?

Reply:
The `k_fifo_get()` can return if `k_fifo_cancel_wait()` is called for it.